<?xml version='1.0' encoding='UTF-8'?>
<workflow xmlns="http://vmware.com/vco/workflow" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://vmware.com/vco/workflow http://vmware.com/vco/workflow/Workflow-v4.xsd" root-name="item2" object-name="workflow:name=generic" id="5d4a74f4-1864-423d-9047-6255c49c9dc8" version="0.0.0" api-version="6.0.0" allowed-operations="vfe" editor-version="2.0" restartMode="1" resumeFromFailedMode="0">
  <display-name><![CDATA[Workflow Missing Throw Exception]]></display-name>
  <description><![CDATA[This workflow illustrates using the Throw Exception schema element.]]></description>
  <position y="65.0" x="50.0"/>
  <input>
    <param name="throwException" type="boolean">
      <description><![CDATA[Do you want to throw an exception?]]></description>
    </param>
  </input>
  <workflow-item name="item0" type="end" end-mode="0">
    <position y="65.0" x="300.0"/>
  </workflow-item>
  <workflow-item name="item1" type="end" end-mode="0">
    <in-binding/>
    <position y="130.0" x="180.0"/>
  </workflow-item>
  <workflow-item name="item2" out-name="item0" type="condition" alt-out-name="item1" comparator="0">
    <display-name><![CDATA[Throw Exception?]]></display-name>
    <script encoded="false"><![CDATA[// Generated by the system, cannot be edited
return (throwException === true);]]></script>
    <in-binding>
      <bind name="throwException" type="boolean" export-name="throwException"/>
    </in-binding>
    <out-binding/>
    <condition name="throwException" type="boolean" comparator="0" label="null"/>
    <description><![CDATA[Custom decision based on a custom script.]]></description>
    <position y="65.0" x="140.0"/>
  </workflow-item>
  <presentation/>
</workflow>